README.md

```diff
@@ -58,6 +58,6 @@ This repository forms part of the Eiffel Community. Eiffel is a protocol for tec
 Visit [Eiffel Community](https://eiffel-community.github.io) to get started and get involved.
 
 # Documentation
-1. [**Hosting Environments**](./wiki/markdown/Hosting_Environments.md)
-2. [**Getting Started**](./wiki/markdown/Getting_Started.md)
-3. [**Bundles**](./wiki/markdown/Bundles.md)
+1. [**Hosting Environments**](wiki/Hosting_Environments.md)
+2. [**Getting Started**](wiki/Getting_Started.md)
+3. [**Bundles**](wiki/Bundles.md)
```

Comment:
Shouldn't these links be prepended with './'?

Reply:
Yes they should, my editor is messing with me ...

Reply:
Good!
You didn't change anything in the bundles docs or tutorials other than some structure, right? Not that you're not allowed to, but I believe it is best to first restructure and then update text in a new PR if needed.

Reply:
Yes, just moved around existing information and fixed some spelling mistakes. Nothing new and nothing removed (despite what Git diff shows)

Reply:
Approved.
@tobiasake , please clarify the Helm init command documentation in a new PR if you see the need for it.